src/common/data/clarity-js-ref.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from 'fs';
+import renderToString from 'next-mdx-remote/render-to-string';
+
+export const convertClarityJSRefToMdx = async () => {
+  const md = fs.readFileSync('./src/_data/clarity-js/globals.md');
```

Comment:
I'm seeing a `Error: ENOENT: no such file or directory, open './src/_data/clarity-js/globals.md'` - you are probably already aware bc this is a draft PR

Reply:
@agraebe  I couldn't commit the md files because of lint prettier issues.

Reply:
`lint:prettier:fix` did not help

Reply:
I have added the files now